ci/builders/README.md

```diff
@@ -297,7 +297,12 @@ default is empty which means no interpreter will be used to run the script
 and it is assumed the script is already an executable with the right
 permissions to run in the target platform.
 * **name** - the name of the step running the script.
-* **parameters** - flags or parameters passed to the script.
+* **parameters** - flags or parameters passed to the script. Parameters
+accept magic environment variables(placeholders replaced before executing
+the test). Magic environment variables have the following limitations:
+only `${FLUTTER_LOGS_DIR}` is currently supported and it needs to be used
+alone within the parameter string(e.g. `["${FLUTTER_LOGS_DIR}"]` is OK
+but `["path=${FLUTTER_LOGS_DIR}"]` is not).
```

Comment:
Curious about the reason for this restriction. Could you point me at the implementation?

Reply:
The restriction is not technical, it was just to simplify the initial implementation. https://cs.opensource.google/flutter/recipes/+/main:recipes/engine_v2/builder.py;l=101

Reply:
kk. Is there an issue filed to follow-up on that? Looping over the keys and doing a find/replace seems like a small, easy improvement.

Reply:
Created a tracking bug: https://github.com/flutter/flutter/issues/129469